bin/helm-include.sh

```diff
@@ -95,8 +95,39 @@ function helmRepoAdd {
   fi
 }
 
+function get_helmchart_reference {
+  local chart_repository chart_name chart_version
+
+  chart_repository=$1
+  chart_name=$2
+  chart_version=$3
+
+  if [ -z "$chart_repository" ]; then
+     echo "'ERROR: Helm chart repository not specified'"
+     return 1
+  elif [ -z "$chart_name" ]; then
+     echo "'ERROR: Helm chart name not specified'"
+     return 1
+  elif [ -z "$chart_version" ]; then
+     echo "'ERROR: Helm chart version not specified'"
+     return 1
+  else
+     #all parms have values
+     :
+  fi
+
+  if [ "$AIRGAP_HELM_FORMAT"  == "tgz" ]; then
+      echo "${AIRGAP_HELM_REPO}${chart_name}-${chart_version}.tgz"
+  elif [ "$AIRGAP_HELM_FORMAT"  == "oci" ]; then
+      echo "oci://${AIRGAP_HELM_REPO}/${chart_repository}/${chart_name}"
```

Comment:
I moved the prefixing with "oci://" to here and explicitly insert the "/" between the AIRGAP_HELM_REPO and the first part of the chart name.  While this can be considered a breaking change, I think the increased in usability outweigh the downsides, especially since the previous/existing functionality has only been available for a single release cycle.

Reply:
I agree.  I'm okay with this change, as well.  I don't think it will necessarily be breaking because the TGZ option would not have existed in the previous iteration and I would expect the deployment to work just fine.